### PR TITLE
Add "Name" in FontConfig

### PR DESCRIPTION
**Pull request by dumkin**
I suggest adding each font its name.
This is done so that in one folder "fontconfigs" (example) you could store the settings for several fonts at once. When loading and saving, you will need to specify the font name in FontConfig

### CODE REVIEW

**Comment by 4ydx** [comment]
Thanks!

File: truetype.go [hunk 1]
```diff
@@ -152,8 +152,10 @@ func NewTruetypeFontConfig(r io.Reader, scale fixed.Int26_6, runeRanges RuneRang
 	return fc, nil
 }
 
-func LoadTruetypeFontConfig(rootPath string) (*FontConfig, error) {
-	fc := &FontConfig{}
+func LoadTruetypeFontConfig(rootPath, name string) (*FontConfig, error) {
+	fc := &FontConfig{} 
+	fc.Name = name;
```
Comment (dmitshur) [inline truetype.go:157]: Semicolon isn't needed. Running `gofmt` would remove it.

Comment (4ydx) [reply truetype.go:157]: Thanks @shurcooL.

**Comment by 4ydx** [comment]
@Dumkin I don't know how you are editing your code, but I assume that there is probably a way to set up a gofmt call when you are saving your file.  That is what I do in neovim.

**Comment by dumkin** [comment]
@4ydx I'm sorry, habit. On account of gofmt many thanks, I'm new to golang, did not know this feature, I'll definitely install

**Pull request merged**